### ISSUE TITLE
Mistral

### ISSUE THREAD

**Issue by fakerybakery** [LargeWorldModel/LWM#10]
Hi,
LWM is incredible! Any plans to release a Mistral version?
Thanks!

**Comment by lhao499** [comment]
Hi, thanks for your interest! We don't have a concrete timeline to add a Mistral version yet, as we are currently occupied with other priorities. However, we will keep this in mind.

**Comment by michaelwechner** [comment]
Sorry for maybe asking a stupid question, but what do you mean with a Mistral version of the LWM?

**Comment by fakerybakery** [comment]
> Sorry for maybe asking a stupid question, but what do you mean with a Mistral version of the LWM?

Yeah

**Comment by michaelwechner** [comment]
> > Sorry for maybe asking a stupid question, but what do you mean with a Mistral version of the LWM?
> 
> Yeah

Do I understand correctly that the LWM is currently built upon "Llama2 7B" and that you would like to have a LWM built upon "Mistral 7B"?

**Comment by fakerybakery** [comment]
> > > Sorry for maybe asking a stupid question, but what do you mean with a Mistral version of the LWM?
> 
> > 
> 
> > Yeah
> 
> 
> 
> Do I understand correctly that the LWM is currently built upon "Llama2 7B" and that you would like to have a LWM built upon "Mistral 7B"?

Correct! Llama is not technically open source due to its license, but Mistral is. Also, Mistral performs much better than Llama

**Comment by KBB99** [comment]
Has anyone made progress with building this on Mistral?
